### ISSUE TITLE
Windows dll - API version incorrect

### ISSUE THREAD

**Issue by ochristensen** [nLabs-nScope/nScopeAPI#2]
I'm trying to do a wrapper for the API in .NET (using the Windows dll) and I'm seeing some oddities (I do recognize this is very much pre-release API!):
1) nScope_check_API_version returns 0.5 not 0.6 (nScope_check_FW_version returns 0.6)
2) nScope_open(1) does indeed power on the nScope (woo!) but it always returns a null pointer...which leaves me stuck without being able to test anything else.

```
//NSCOPE_API_EXPORT_CALL scopeHandle* nScope_open(int powerOn);
[DllImport("libnscope.dll", CallingConvention = CallingConvention.Cdecl)]
public static extern IntPtr nScope_open(int powerOn);
```

**Comment by theotherplanb** [comment]
I think I am seeing the same issue, testing with the Python API and the Windows dll. Although the commit is labeled "0.6", `checkAPIver()` returns `0.5` and if I try to initialize the nScope, I get

```
Version Mismatch:
API Version: 0.50
FW  Version: 0.60

Unable to connect to nScope
```

**Comment by davidjmeyer** [comment]
Hm, I may have forgotten to build the DLL version for release 0.6, it was a rather time-pressured release. 

Maybe you can try linking against the .lib file?

Unfortunately, my windows build environment is down right now, so it may be a few days before I can figure out what's going on.  I'll let you know.

**Comment by davidjmeyer** [comment]
Actually, I got the build up faster than I thought I would. Should be fixed in commit cf0da78
